router/router.go

```diff
@@ -73,6 +85,11 @@ func NewJsonDirectoryServer(schemaDirectory string, validator openrtb_ext.Bidder
 		data[bidder] = json.RawMessage(validator.Schema(bidderName))
 	}
 
+	//Add in any aliases
```

Comment:
nit add space after `//`
`//Add in any aliases` -> `// Add in any aliases`

Reply:
https://github.com/prebid/prebid-server/pull/3055/commits/d5c207f18d210f765ed3301ddcc857da973a8b60